docs/pipelines/tasks/deploy/azure-resource-group-deployment.md

```diff
@@ -53,6 +53,45 @@ Use this task to deploy, start, stop, and delete Azure Resource Groups.
 |`deploymentOutputs`<br/>Deployment outputs|(Optional) Provide a name for the variable for the output variable which will contain the outputs section of the current deployment object in string format. You can use the **ConvertFrom-Json** PowerShell cmdlet to parse the JSON object and access the individual output values.|
 |`addSpnToEnvironment`<br/>Access service principal details in override parameters| Adds service principal ID and key of the Azure endpoint you chose to the script's execution environment. You can use these variables: **$servicePrincipalId** and **$servicePrincipalKey** in your override parameters like **-key $servicePrincipalKey**|
 
+## Troubleshooting
+
+## Error: Internal Server Error
+
+These issues are mostly transient in nature. There are multiple reasons why it could be happening:
+1. One of the Azure service you're trying to deploy is undergoing maintainance in the region you're trying to deploy to. Keep an eye out on https://status.azure.com/ to check downtimes of Azure Services.
+2. Azure Pipelines service itself is going through maintenance. Keep an eye out on https://status.dev.azure.com/ for downtimes.
+
+However, we've seen some instances where this is due to an error in the ARM template, such as:
+* Azure service you're trying to deploy doesn't support the region you've chosen for the resource.
+
+## Error: Timeout
+
+Timeout issues could be coming from two places:
```

Comment:
Use the markdown number format like 
1. Azure ...
1. Something

Reply:
Yes that's how it looks right now